wildfly-elytron/pom.xml

```diff
@@ -915,6 +906,14 @@
                                     <include>org.wildfly.security.x500.cert</include>
                                 </includes>
                                 <excludes>
+                                    <exclude>org.wildfly.security.x500.cert.ElytronMessages_$logger</exclude>
```

Comment:
It looks like this exclusion can be removed.

Reply:
removed